build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
 
     ext {
         versions = [
-                'androidGradlePlugin': '4.2.0-beta06',      // https://developer.android.com/studio/releases/gradle-plugin.html
+                'androidGradlePlugin': '4.2.0-rc01',      // https://developer.android.com/studio/releases/gradle-plugin.html
```

Comment:
We can bump this straight to `4.2.0` now.

Reply:
I'll update this